.idea/runConfigurations/All_unit_tests.xml

```diff
@@ -5,6 +5,7 @@
     <node-options value="" />
     <jest-package value="$PROJECT_DIR$/node_modules/jest" />
     <working-dir value="$PROJECT_DIR$" />
+    <jest-options value="--watch" />
```

Comment:
What does this do?

Reply:
This flag turn on test runner in `watch` mode, that's means that all changes of test files rerun tests